README.md

```diff
@@ -59,7 +59,7 @@ This Repository is for the NHLBI BDC Metrics Dashboard. A proof of Concept of th
 ### Exports: 
 * Dug: N/A
 * PIC-SURE: The number of times a cohort has been Exported to Seven Bridges, Exported to Terra, or downloaded as either a CSV or PFB
-* Gen3: The number of times a cohort has been exported to CGC, CAVATICA, BDC (Seven Bridges), Terra, or exported as a PFB
+* Gen3: The number of times a cohort has been exported as a PFB
```

Comment:
Currently, Gen3 uses a frontend redirect to export to an analysis platform, which is outside or observability boundary, so we can not provide targets of export right now.